modules/ossm-extensions-ref-smextension.adoc

```diff
@@ -0,0 +1,45 @@
+////
+This module included in the following assemblies:
+*service_mesh_/v2x/ossm-extensions.adoc
```

Comment:
I still think this section should start with a [big] note/warning that everything below this point is deprecated. When reading the docs, this looks like just another section that looks a bit duplicate from what we read above. It's a bit confusing.

If this is not possible, what about at least mention `[deprecated]` every time `ServiceMeshExtension` is mentioned in a section title from now on?

Reply:
Our doc tooling and standards have some restrictions around how I can handle this without breaking things.  I've added a note about deprecation to each of the ServiceMeshExtension topics.